robber/matchers/contain.py

```diff
@@ -5,16 +5,33 @@
 
 class Contain(Base):
     """
-    expect({'key': value}).to.contain('key')
-    expect([1, 2, 3]).to.contain(2)
+    expect({'key': value}).to.contain('key 1', 'key 2', 'key n')
+    expect([1, 2, 3]).to.contain(1, 2, 3)
     """
 
     def matches(self):
-        return self.expected in self.actual
+        expected_list = list(self.args)
+        expected_list.insert(0, self.expected)
+
+        if not self.is_negative:
+            for expected in expected_list:
+                if expected not in self.actual:
+                    self.expected_arg = expected
+                    return False
+
+            return True
+        else:
+            # As this is the negative case, we have to flip the return value.
+            for expected in expected_list:
```

Comment:
I guess we can put this loop into a function and return its negative form, the result is still the same, right?
```
def is_contained(expected, expecteds):
  ..

contained = is_contained(expected, expecteds)
return contained if self.is_negative else !contained
```